### DISCUSSION TITLE
adding hardware (ledger) wallet addresses

### DISCUSSION THREAD

**Discussion by albist** [rainbow-me/rainbowkit#1529] [Q&A] [answered]
hey,
when adding a wallet from my ledger, I can not chose the addresses I want to import.
each time the two same addresses are imported.
I would like to chose which of the ledger addresses I want to import on rainbow.
am I missing something ?

**Comment by DanielSinclair** [comment]
@albist Which Rainbow product are you referring to? The mobile app or the browser extension?

**Reply by albist** [reply]
hey, sorry, I refer to the mobile app.

**Reply by DanielSinclair** [reply, marked as answer]
@albist Thanks for clarifying. I've relayed the issue to our App team. Please also send an email to `support@rainbow.me` for further follow-up questions and to get notified when this is fixed. Closing here, but we'll work on a fix!
